metakube/resource_node_deployment_schema.go

```diff
@@ -335,6 +335,17 @@ func matakubeResourceNodeDeploymentCloudOpenstackSchema() map[string]*schema.Sch
 			Description:      "Specifies how long should the controller check if instance is ready before timing out",
 			ValidateDiagFunc: isNonEmptyDurationString,
 		},
+		"server_group_id": {
+			Type:     schema.TypeString,
+			Optional: true,
+			Default:  "",
+			DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
```

Comment:
could we maybe add a test where we create a md without sg, make a change but not to sg, and make another change where you set the sg explicitely?